libs/coin-modules/coin-sui/src/network/sdk.ts

```diff
@@ -37,7 +40,21 @@ export async function withApi<T>(execute: AsyncApiFunction<T>) {
   const url = coinConfig.getCoinConfig().node.url;
 
   if (!apiMap[url]) {
-    apiMap[url] = new SuiClient({ url });
+    // Create a ky instance with extended timeout and retry logic
+    const kyExtendedTimeout = ky.create({
+      timeout: 60000,
+      headers: [["X-Ledger-Client-Version", getEnv("LEDGER_CLIENT_VERSION")]],
```

Comment:
Shouldn't this be ...?

```suggestion
      headers: { "X-Ledger-Client-Version", getEnv("LEDGER_CLIENT_VERSION") },
```

Reply:
Or is it the same? It might be easier to read in the future in case you have to add more headers

Reply:
it's the same, it was linter suggestion to use tuple instead of object, because key has dashes